ArmoniK.SDK.DynamicWorker/CMakeLists.txt

```diff
@@ -48,4 +71,4 @@ install(TARGETS ${PROJECT_NAME}
         RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
         ARCHIVE DESTINATION ${CMAKE_INSTALL_BINDIR})
 
-install(FILES appsettings.json DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(FILES appsettings.json DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
```

Comment:
It should mean the same thing

Reply:
Yes, it is the same thing, but if the folder is not yet created CMake generation fails for FILES copies. This is why I choose the second option.

Reply:
Yes, there are totally equal!
The problem is when I use ${CMAKE_INSTALL_BINDIR} in install(FILES appsettings.json DESTINATION ${CMAKE_INSTALL_PREFIX}/bin) I will get an error if the bin directory is not yet created. The install file does not force the creation of the directory while the install target forces it.